pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/SegmentIndexCreationDriverImpl.java

```diff
@@ -113,6 +112,29 @@ public class SegmentIndexCreationDriverImpl implements SegmentIndexCreationDrive
   private long _totalStatsCollectorTime = 0;
   private boolean _continueOnError;
 
+  public static void persistCreationMeta(File indexDir, long crc, long creationTime)
```

Comment:
These changes are not related to this PR. I guess you might enabled `Rearrange code` option in IDE during auto reformat. Can you revert these unrelated changes? There are quite some unrelated auto reformat in several files

Reply:
Yeah. I will look into this formatting issue.